src/pal/src/debug/debug.cpp

```diff
@@ -343,15 +344,18 @@ DebugBreakCommand()
     const char *command_string = getenv (PAL_RUN_ON_DEBUG_BREAK);
     if (command_string) {
         char pid_buf[sizeof (PID_TEXT) + 32];
-        char exe_buf[sizeof (EXE_TEXT) + MAX_LONGPATH + 1];
+		    PathCharString exe_bufString;
```

Comment:
@Priya91, this repo has editorconfig at the root: https://github.com/dotnet/coreclr/blob/master/.editorconfig. If you install editorconfig plugin/extension for your IDE/editor (available for almost all known to mankind), you won't get any formatting problem: http://editorconfig.org/.

Bonus; in VS, editorconfig extension does not permanently change your editor settings but only scope it to the opened solution which has `.editorconfig` file available at root.